Add some channels

BellezaEmporium opened the pull request:
Closes https://github.com/iptv-org/iptv/issues/4031

BellezaEmporium committed:
TN : Add some channels

BellezaEmporium committed:
Update uk.m3u

BellezaEmporium committed:
Update eg.m3u

BellezaEmporium committed:
Update eg.m3u

BellezaEmporium committed:
Update eg.m3u

BellezaEmporium committed:
Update eg.m3u

BellezaEmporium committed:
Update eg.m3u

BellezaEmporium committed:
Update kw.m3u

Dum4G commented on channels/uk.m3u:
```diff
@@ -15,6 +15,8 @@ https://cdnamd-hls-globecast.akamaized.net/live/ramdisk/akaal_tv/hls1_smart_akaa
 #EXTINF:-1 tvg-id="AlarabyTV.uk" tvg-name="Alaraby TV" tvg-country="ARAB" tvg-language="Arabic" tvg-logo="https://i.imgur.com/LNI1HNi.png" group-title="News",Alaraby TV (1080p)
 https://alaraby.cdn.octivid.com/alaraby/smil:alaraby.stream.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="AlMostaquilla.uk" tvg-name="Al Mostaquilla" tvg-country="ARAB" tvg-language="Arabic" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/b/b0/Almostaqila.jpg" group-title="News",Al Mustaquilla
+http://192.34.153.40:7777/AlMustakila.m3u8?user=sgls540839&session=f02296b319bb8e1df11e8baa497e110421c4dea2c4cd081e480ec0588335ddb9ba2d1728ee76dd0c384e97fef14771cc
```
Drops connection to me

jkemp814 commented:
![image](https://user-images.githubusercontent.com/12059343/128641638-216544f5-a867-4aed-bdd8-40abd9faf6ab.png)

BellezaEmporium committed:
Merge branch 'master' into LaneSh4d0w-patch-1

LaneSh4d0w commented:
Indeed, it was working fine for me a few hours ago.

LaneSh4d0w commented:
I'll use my second card, wait.

LaneSh4d0w commented:
Found the link from http://www.elahmad.com/tv/arabic-tv-online.php?id=almustakila&type=flash

Their code is so "well-written" it's giving me harsh times getting the lin out of it (and they've encoded it inside a JWT token... that's given *after* everything had started)

BellezaEmporium committed:
Merge branch 'master' into LaneSh4d0w-patch-1

BellezaEmporium committed:
Add Filmon version of Penthouse

Can't say if this is a real stream or just a compilation of videos. Saying it like so.

LaneSh4d0w commented:
http://www.filmon.com/vr-streams/6200.high/playlist.m3u8 PBS but can't tell from which state this is streamed from.

BellezaEmporium committed:
Update uk.m3u

BellezaEmporium committed:
Update eg.m3u

BellezaEmporium committed:
Update kw.m3u

Dum4G commented on channels/kw.m3u:
```diff
@@ -21,22 +19,14 @@ http://www.elahmad.com/tv/m3u8/alkass.m3u8?id=alkass3
 #EXTINF:-1 tvg-id="ATV.kw" tvg-name="ATV" tvg-country="KW" tvg-language="Arabic" tvg-logo="" group-title="",ATV (360p)
 https://gulfsat.cdn.easybroadcast.fr/live/Aladalah_abr/playlist.m3u8
 #EXTINF:-1 tvg-id="atv.kw" tvg-name="atv" tvg-country="KW" tvg-language="Arabic" tvg-logo="http://www.atvkuwait.com/img/logo/atv_logo.png" group-title="",atv
 rtmp://168.187.49.14:1935/live//atv?v=
```
looks dead to me

LaneSh4d0w replied:
Gonna check later

tilfizyawn replied:
dead for me too

LaneSh4d0w replied:
Dead indeed.

spotanjo3 commented:
`https://iptv-org.github.io/iptv/countries/pt.m3u`

This is Portugal Channel. Can you add:

Sport TV 1 HD
Sport TV 2 HD
Sporting TV HD
BTV 1 HD 
TVI24 HD
STV Notícias
SIC Notícias HD

Please ? 


<br class="Apple-interchange-newline" style="caret-color: rgb(0, 0, 0); color: rgb(0, 0, 0); font-style: normal; font-variant-caps: normal; font-weight: normal; letter-spacing: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; text-decoration: none;">

tilfizyawn commented on channels/kw.m3u:
```diff
@@ -5,10 +5,8 @@ http://www.elahmad.com/tv/m3u8/alkass.m3u8
 http://38.96.175.40:1935/liveTrans/ngrp:channel23_all/playlist.m3u8
 #EXTINF:-1 tvg-id="ALRAI.kw" tvg-name="AL RAI" tvg-country="KW" tvg-language="" tvg-logo="" group-title="",AL RAI
```
```suggestion
#EXTINF:-1 tvg-id="ALRAI.kw" tvg-name="AL RAI" tvg-country="KW" tvg-language="Arabic" tvg-logo="" group-title="",AL RAI
https://media.streambrothers.com:1936/8724/8724/playlist.m3u8
#EXTINF:-1 tvg-id="ALRAI.kw" tvg-name="AL RAI" tvg-country="KW" tvg-language="Arabic" tvg-logo="" group-title="",AL RAI
```

tilfizyawn replied:
Add official stream from https://www.alraimedia.com/live

BellezaEmporium committed:
Update channels/kw.m3u

Co-authored-by: tilfizyawn <82530182+tilfizyawn@users.noreply.github.com>

BellezaEmporium committed:
Update kw.m3u

BellezaEmporium committed:
took down elahmad streams + clean

1) It's only showing a picture for elahmad streams
2) Delete non working channels.

BellezaEmporium committed:
Merge branch 'master' into LaneSh4d0w-patch-1

LaneSh4d0w commented:
> 
> 
> `https://iptv-org.github.io/iptv/countries/pt.m3u`
> 
> This is Portugal Channel. Can you add:
> 
> Sport TV 1 HD
> Sport TV 2 HD
> Sporting TV HD
> BTV 1 HD
> TVI24 HD
> STV Notícias
> SIC Notícias HD
> SIC Internacional
> TVI Internacional
> 
> Please ?

Sport TV 1 https://sport.livedoomovie.com/02_SPORTTV_1_720p/chunklist.m3u8
Sport TV 2 https://sport.livedoomovie.com/02_SPORTTV_2_720p/chunklist.m3u8
BTV 1 HD -> Benfica TV : no
TVI24 HD https://iptv-all.lanesh4d0w.codes/portugal/tvi24

jkemp814 commented:
Sport TV 1 https://sport.livedoomovie.com/02_SPORTTV_1_720p/chunklist.m3u8 - (will not connect from here)
Sport TV 2 https://sport.livedoomovie.com/02_SPORTTV_2_720p/chunklist.m3u8 - (will not connect from here)

LaneSh4d0w commented:
> Sport TV 1 https://sport.livedoomovie.com/02_SPORTTV_1_720p/chunklist.m3u8 - (will not connect from here)
> Sport TV 2 https://sport.livedoomovie.com/02_SPORTTV_2_720p/chunklist.m3u8 - (will not connect from here)
> 
> 

There's a weird fix that @Dum4G found for those Livedoo links.

spotanjo3 commented:
It doesnt work. Dont you add Sport TV 1 and Sport TV 2 to your PT m3u8 ?
And keep up the good work! I hope you resolve it.

On Mon, Aug 9, 2021 at 3:21 PM Shadix A. ***@***.***> wrote:

> Sport TV 1 https://sport.livedoomovie.com/02_SPORTTV_1_720p/chunklist.m3u8
> - (will not connect from here)
> Sport TV 2 https://sport.livedoomovie.com/02_SPORTTV_2_720p/chunklist.m3u8
> - (will not connect from here)
>
> There's a weird fix that @Dum4G <https://github.com/Dum4G> found for
> those Livedoo links.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/iptv-org/iptv/pull/4000#issuecomment-895478121>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AD5X34RYM5BMSSG6AZGYZGDT4ATCNANCNFSM5BYUVICQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&utm_campaign=notification-email>
> .
>


-- 
In Memory of Spot (FOREVER)

spotanjo3 commented:
> > `https://iptv-org.github.io/iptv/countries/pt.m3u`
> > This is Portugal Channel. Can you add:
> > Sport TV 1 HD
> > Sport TV 2 HD
> > Sporting TV HD
> > BTV 1 HD
> > TVI24 HD
> > STV Notícias
> > SIC Notícias HD
> > SIC Internacional
> > TVI Internacional
> > Please ?
> 
> Sport TV 1 https://sport.livedoomovie.com/02_SPORTTV_1_720p/chunklist.m3u8
> Sport TV 2 https://sport.livedoomovie.com/02_SPORTTV_2_720p/chunklist.m3u8
> Sporting TV HD no
> BTV 1 HD -> Benfica TV no
> TVI24 HD https://iptv-all.lanesh4d0w.codes/portugal/tvi24
> STV Noticias http://dcunilive36-lh.akamaihd.net/i/dclive_1@668002/master.m3u8
> SIC Noticias http://live.impresa.pt/live/sicnot/sicnot.m3u8 Geo-locked
> SIC International http://live.impresa.pt/live/sicint/sicint.m3u8
> TVI Internacional Login or subscription required.

Sport TV 1 https://sport.livedoomovie.com/02_SPORTTV_1_720p/chunklist.m3u8 ---- NOT WORKING
Sport TV 2 https://sport.livedoomovie.com/02_SPORTTV_2_720p/chunklist.m3u8 ------ NOT WORKING
Sporting TV HD no
BTV 1 HD -> Benfica TV no
TVI24 HD https://iptv-all.lanesh4d0w.codes/portugal/tvi24  ----- WORKING
STV Noticias http://dcunilive36-lh.akamaihd.net/i/dclive_1@668002/master.m3u8 ----- WORKING
SIC Noticias http://live.impresa.pt/live/sicnot/sicnot.m3u8 Geo-locked ----- NOT WORKING
SIC International http://live.impresa.pt/live/sicint/sicint.m3u8 ----- WORKING

Please add them to PT.M3u8 then. And hope you find more working as above. Cheers.

LaneSh4d0w commented:
> 
> 
> > > `https://iptv-org.github.io/iptv/countries/pt.m3u`
> > > This is Portugal Channel. Can you add:
> > > Sport TV 1 HD
> > > Sport TV 2 HD
> > > Sporting TV HD
> > > BTV 1 HD
> > > TVI24 HD
> > > STV Notícias
> > > SIC Notícias HD
> > > SIC Internacional
> > > TVI Internacional
> > > Please ?
> > 
> > 
> > Sport TV 1 https://sport.livedoomovie.com/02_SPORTTV_1_720p/chunklist.m3u8
> > Sport TV 2 https://sport.livedoomovie.com/02_SPORTTV_2_720p/chunklist.m3u8
> > Sporting TV HD
> > BTV 1 HD -> Benfica TV no
> > TVI24 HD https://iptv-all.lanesh4d0w.codes/portugal/tvi24
> > STV Noticias http://dcunilive36-lh.akamaihd.net/i/dclive_1@668002/master.m3u8
> > SIC Noticias http://live.impresa.pt/live/sicnot/sicnot.m3u8 Geo-locked
> > SIC International http://live.impresa.pt/live/sicint/sicint.m3u8
> > TVI Internacional Login or subscription required.
> 
> Sport TV 1 https://sport.livedoomovie.com/02_SPORTTV_1_720p/chunklist.m3u8 ---- NOT WORKING
> Sport TV 2 https://sport.livedoomovie.com/02_SPORTTV_2_720p/chunklist.m3u8 ------ NOT WORKING
> Sporting TV HD no -- Hope you find one.
> BTV 1 HD -> Benfica TV no ---- Hope you find one.
> TVI24 HD https://iptv-all.lanesh4d0w.codes/portugal/tvi24 ----- WORKING
> STV Noticias http://dcunilive36-lh.akamaihd.net/i/dclive_1@668002/master.m3u8 ----- WORKING
> SIC Noticias http://live.impresa.pt/live/sicnot/sicnot.m3u8 Geo-locked ----- NOT WORKING
> SIC International http://live.impresa.pt/live/sicint/sicint.m3u8 ----- WORKING
> 
> Please add them to PT.M3u8 then. And hope you find more working as above. Cheers.

![image](https://user-images.githubusercontent.com/30985701/128775044-3cd5cc8d-7738-411f-83f9-1a60f0aee695.png)
![image](https://user-images.githubusercontent.com/30985701/128775078-1b1243b1-7c98-4fa5-88e1-d7d1caba1a15.png)

spotanjo3 commented:
What player do you use ? I test them on VLC Player.

On Mon, Aug 9, 2021 at 5:12 PM Shadix A. ***@***.***> wrote:

> https://iptv-org.github.io/iptv/countries/pt.m3u
> This is Portugal Channel. Can you add:
> Sport TV 1 HD
> Sport TV 2 HD
> Sporting TV HD
> BTV 1 HD
> TVI24 HD
> STV Notícias
> SIC Notícias HD
> SIC Internacional
> TVI Internacional
> Please ?
>
> Sport TV 1 https://sport.livedoomovie.com/02_SPORTTV_1_720p/chunklist.m3u8
> Sport TV 2 https://sport.livedoomovie.com/02_SPORTTV_2_720p/chunklist.m3u8
> Sporting TV HD
> BTV 1 HD -> Benfica TV no
> TVI24 HD https://iptv-all.lanesh4d0w.codes/portugal/tvi24
> STV Noticias
> ***@***.***/master.m3u8
> SIC Noticias http://live.impresa.pt/live/sicnot/sicnot.m3u8 Geo-locked
> SIC International http://live.impresa.pt/live/sicint/sicint.m3u8
> TVI Internacional Login or subscription required.
>
> Sport TV 1 https://sport.livedoomovie.com/02_SPORTTV_1_720p/chunklist.m3u8
> ---- NOT WORKING
> Sport TV 2 https://sport.livedoomovie.com/02_SPORTTV_2_720p/chunklist.m3u8
> ------ NOT WORKING
> Sporting TV HD no -- Hope you find one.
> BTV 1 HD -> Benfica TV no ---- Hope you find one.
> TVI24 HD https://iptv-all.lanesh4d0w.codes/portugal/tvi24 ----- WORKING
> STV Noticias
> ***@***.***/master.m3u8 -----
> WORKING
> SIC Noticias http://live.impresa.pt/live/sicnot/sicnot.m3u8 Geo-locked
> ----- NOT WORKING
> SIC International http://live.impresa.pt/live/sicint/sicint.m3u8 -----
> WORKING
>
> Please add them to PT.M3u8 then. And hope you find more working as above.
> Cheers.
>
> [image: image]
> <https://user-images.githubusercontent.com/30985701/128775044-3cd5cc8d-7738-411f-83f9-1a60f0aee695.png>
> [image: image]
> <https://user-images.githubusercontent.com/30985701/128775078-1b1243b1-7c98-4fa5-88e1-d7d1caba1a15.png>
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/iptv-org/iptv/pull/4000#issuecomment-895553070>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AD5X34RR7OSYXGNRSJS6EQ3T4BADXANCNFSM5BYUVICQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&utm_campaign=notification-email>
> .
>


-- 
In Memory of Spot (FOREVER)

jkemp814 commented:
Which two channels are those @LaneSh4d0w ?

LaneSh4d0w commented:
> 
> 
> Which two channels are those @LaneSh4d0w ?

Sport TV 1 / Sport TV 2

LaneSh4d0w commented:
> 
> What player do you use ? I test them on VLC Player.
> […](#)
> On Mon, Aug 9, 2021 at 5:12 PM Shadix A. ***@***.***> wrote: https://iptv-org.github.io/iptv/countries/pt.m3u This is Portugal Channel. Can you add: Sport TV 1 HD Sport TV 2 HD Sporting TV HD BTV 1 HD TVI24 HD STV Notícias SIC Notícias HD SIC Internacional TVI Internacional Please ? Sport TV 1 https://sport.livedoomovie.com/02_SPORTTV_1_720p/chunklist.m3u8 Sport TV 2 https://sport.livedoomovie.com/02_SPORTTV_2_720p/chunklist.m3u8 Sporting TV HD BTV 1 HD -> Benfica TV no TVI24 HD https://iptv-all.lanesh4d0w.codes/portugal/tvi24 STV Noticias ***@***.***/master.m3u8 SIC Noticias http://live.impresa.pt/live/sicnot/sicnot.m3u8 Geo-locked SIC International http://live.impresa.pt/live/sicint/sicint.m3u8 TVI Internacional Login or subscription required. Sport TV 1 https://sport.livedoomovie.com/02_SPORTTV_1_720p/chunklist.m3u8 ---- NOT WORKING Sport TV 2 https://sport.livedoomovie.com/02_SPORTTV_2_720p/chunklist.m3u8 ------ NOT WORKING Sporting TV HD no -- Hope you find one. BTV 1 HD -> Benfica TV no ---- Hope you find one. TVI24 HD https://iptv-all.lanesh4d0w.codes/portugal/tvi24 ----- WORKING STV Noticias ***@***.***/master.m3u8 ----- WORKING SIC Noticias http://live.impresa.pt/live/sicnot/sicnot.m3u8 Geo-locked ----- NOT WORKING SIC International http://live.impresa.pt/live/sicint/sicint.m3u8 ----- WORKING Please add them to PT.M3u8 then. And hope you find more working as above. Cheers. [image: image] <https://user-images.githubusercontent.com/30985701/128775044-3cd5cc8d-7738-411f-83f9-1a60f0aee695.png> [image: image] <https://user-images.githubusercontent.com/30985701/128775078-1b1243b1-7c98-4fa5-88e1-d7d1caba1a15.png> — You are receiving this because you commented. Reply to this email directly, view it on GitHub <[#4000 (comment)](https://github.com/iptv-org/iptv/pull/4000#issuecomment-895553070)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AD5X34RR7OSYXGNRSJS6EQ3T4BADXANCNFSM5BYUVICQ> . Triage notifications on the go with GitHub Mobile for iOS <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android <https://play.google.com/store/apps/details?id=com.github.android&utm_campaign=notification-email> .
> -- In Memory of Spot (FOREVER)

They work with a specific user-agent on VLC. I'll need to look again at the user agent Dum4G had given us.

spotanjo3 commented:
Yeah ? I used VLC and I tried it again and those two wont work. Yep, I am
using the latest version of VLC.

On Mon, Aug 9, 2021 at 5:17 PM Shadix A. ***@***.***> wrote:

> What player do you use ? I test them on VLC Player.
> … <#m_5852700084877696155_>
> On Mon, Aug 9, 2021 at 5:12 PM Shadix A. *@*.*> wrote:
> https://iptv-org.github.io/iptv/countries/pt.m3u
> <https://iptv-org.github.io/iptv/countries/pt.m3u> This is Portugal
> Channel. Can you add: Sport TV 1 HD Sport TV 2 HD Sporting TV HD BTV 1 HD
> TVI24 HD STV Notícias SIC Notícias HD SIC Internacional TVI Internacional
> Please ? Sport TV 1
> https://sport.livedoomovie.com/02_SPORTTV_1_720p/chunklist.m3u8
> <https://sport.livedoomovie.com/02_SPORTTV_1_720p/chunklist.m3u8> Sport TV
> 2 https://sport.livedoomovie.com/02_SPORTTV_2_720p/chunklist.m3u8
> <https://sport.livedoomovie.com/02_SPORTTV_2_720p/chunklist.m3u8> Sporting
> TV HD BTV 1 HD -> Benfica TV no TVI24 HD
> https://iptv-all.lanesh4d0w.codes/portugal/tvi24
> <https://iptv-all.lanesh4d0w.codes/portugal/tvi24> STV Noticias @.*/master.m3u8
> SIC Noticias http://live.impresa.pt/live/sicnot/sicnot.m3u8 Geo-locked
> SIC International http://live.impresa.pt/live/sicint/sicint.m3u8 TVI
> Internacional Login or subscription required. Sport TV 1
> https://sport.livedoomovie.com/02_SPORTTV_1_720p/chunklist.m3u8 ---- NOT
> WORKING Sport TV 2
> https://sport.livedoomovie.com/02_SPORTTV_2_720p/chunklist.m3u8 ------
> NOT WORKING Sporting TV HD no -- Hope you find one. BTV 1 HD -> Benfica TV
> no ---- Hope you find one. TVI24 HD
> https://iptv-all.lanesh4d0w.codes/portugal/tvi24 ----- WORKING STV
> Noticias *@*.***/master.m3u8 ----- WORKING SIC Noticias
> http://live.impresa.pt/live/sicnot/sicnot.m3u8 Geo-locked ----- NOT
> WORKING SIC International http://live.impresa.pt/live/sicint/sicint.m3u8
> ----- WORKING Please add them to PT.M3u8 then. And hope you find more
> working as above. Cheers. [image: image]
> https://user-images.githubusercontent.com/30985701/128775044-3cd5cc8d-7738-411f-83f9-1a60f0aee695.png
> [image: image]
> https://user-images.githubusercontent.com/30985701/128775078-1b1243b1-7c98-4fa5-88e1-d7d1caba1a15.png
> — You are receiving this because you commented. Reply to this email
> directly, view it on GitHub <#4000 (comment)
> <https://github.com/iptv-org/iptv/pull/4000#issuecomment-895553070>>, or
> unsubscribe
> https://github.com/notifications/unsubscribe-auth/AD5X34RR7OSYXGNRSJS6EQ3T4BADXANCNFSM5BYUVICQ
> . Triage notifications on the go with GitHub Mobile for iOS
> https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675
> or Android
> https://play.google.com/store/apps/details?id=com.github.android&utm_campaign=notification-email
> .
> -- In Memory of Spot (FOREVER)
>
> They work with a specific user-agent on VLC. I'll need to look again at
> the user agent Dum4G had given us.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/iptv-org/iptv/pull/4000#issuecomment-895556229>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AD5X34V3BPKRYUDXTME72BTT4BAX3ANCNFSM5BYUVICQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&utm_campaign=notification-email>
> .
>


-- 
In Memory of Spot (FOREVER)

LaneSh4d0w commented:
I also have Sport TV 3 and 4 in the same pack. Gonna add them once I've found the user-agent.

spotanjo3 commented:
What about Sport TV 1 and 2 in the same pack ? And why they didnt worked on
my VLC Player when it works fine for you ?

On Mon, Aug 9, 2021 at 5:22 PM Shadix A. ***@***.***> wrote:

> I also have Sport TV 3 and 4 in the same pack. Gonna add them once I've
> found the user-agent.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/iptv-org/iptv/pull/4000#issuecomment-895558974>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AD5X34QBQ7WHD475JDPHCBDT4BBJPANCNFSM5BYUVICQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&utm_campaign=notification-email>
> .
>


-- 
In Memory of Spot (FOREVER)

BellezaEmporium committed:
Update pt.m3u

LaneSh4d0w commented:
> 
> 
> Which two channels are those @LaneSh4d0w? Never mind I see the logo up in the corner.

Try now with the user-agent, from VLC.

BellezaEmporium committed:
Update pt.m3u

spotanjo3 commented:
> > Which two channels are those @LaneSh4d0w? Never mind I see the logo up in the corner.
> 
> Try now with the user-agent, from VLC.

Still 41 channels. And no new additions.

LaneSh4d0w commented:
> > > Which two channels are those @LaneSh4d0w? Never mind I see the logo up in the corner.
> > 
> > Try now with the user-agent, from VLC.
> 
> Still 41 channels. And no new additions.
> 

Those won't be up the main courses until this PR is merged, if you want to test it, you'll need to take the raw m3u file from my PR.

spotanjo3 commented:
I understand. I want
To test it and where? Thanks.

On Mon, Aug 9, 2021 at 5:50 PM Shadix A. ***@***.***> wrote:

> Which two channels are those @LaneSh4d0w <https://github.com/LaneSh4d0w>?
> Never mind I see the logo up in the corner.
>
> Try now with the user-agent, from VLC.
>
> Still 41 channels. And no new additions.
>
> Those won't be up the main courses until this PR is merged, if you want to
> test it, you'll need to take the raw m3u file from my PR.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/iptv-org/iptv/pull/4000#issuecomment-895574455>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AD5X34TKNI5FQA6FWOCQEXLT4BES5ANCNFSM5BYUVICQ>
> .
>
>
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&utm_campaign=notification-email>
> .
>
-- 
In Memory of Spot (FOREVER)

BellezaEmporium committed:
Merge branch 'master' into LaneSh4d0w-patch-1

LaneSh4d0w commented:
https://raw.githubusercontent.com/iptv-org/iptv/f8cda02ebd0aa08bc732e2d810abbbd1e5b99a85/channels/pt.m3u

Dum4G commented on channels/kw.m3u:
```diff
@@ -1,42 +1,14 @@
-https://stream02.fasttelco.net/live/alrai.stream/playlist.m3u8
-#EXTINF:-1 tvg-id="AlRaiHD.kw" tvg-name="Al Rai HD" tvg-country="KW" tvg-language="" tvg-logo="" group-title="",Al Rai HD
-https://stream02.fasttelco.net/live/alrai.stream/chunklist_w420445644.m3u8
+#EXTINF:-1 tvg-id="ALRAI.kw" tvg-name="AL RAI" tvg-country="KW" tvg-language="Arabic" tvg-logo="" group-title="",AL RAI
```
#EXTINF:-1 tvg-id="AlraiTV.kw" tvg-name="Alrai TV" tvg-country="KW" tvg-language="Arabic" tvg-logo="https://i.imgur.com/eqCEXk6.png" group-title="",Alrai TV

Dum4G commented on channels/kw.m3u:
```diff
@@ -1,42 +1,14 @@
-http://www.elahmad.com/tv/m3u8/alkass.m3u8?id=alkass5
-#EXTINF:-1 tvg-id="AlraiTV.kw" tvg-name="Alrai TV" tvg-country="KW" tvg-language="" tvg-logo="" group-title="",Alrai TV
-http://stream02.fasttelco.net/4/pub/asset/28/streams.m3u8?v=1053886998
-#EXTINF:-1 tvg-id="AL Rai ARB" tvg-name="AL Rai ARB" tvg-country="KW" tvg-language="Arabic" tvg-logo="http://f.alrai.tv/TV/Programs/60_Alrai_Tv21_-__RT655x0-_OS492x240-_RD492x240-.png" group-title="",Alrai TV (1080p)
```
#EXTINF:-1 tvg-id="AlraiTV.kw" tvg-name="Alrai TV" tvg-country="KW" tvg-language="Arabic" tvg-logo="https://i.imgur.com/eqCEXk6.png" group-title="",Alrai TV [Offline]

This link is being in other playlists still

LaneSh4d0w replied:
Well... that one refused to answer until now. But it gives me a 404 on chunklist retrieval.

LaneSh4d0w commented:
> 
> 
> http://www.filmon.com/vr-streams/6200.high/playlist.m3u8 PBS but can't tell from which state this is streamed from.

@jkemp814 @MapGuy11 Any idea on the origin of this PBS ?

Dum4G committed:
Update channels/kw.m3u

LaneSh4d0w commented:
@Dum4G If you think that there's something important to change or to add, don't hesitate to directly edit the files from my branch. I'll check the changes after that.

MapGuy11 commented:
> > 
> > 
> > http://www.filmon.com/vr-streams/6200.high/playlist.m3u8 PBS but can't tell from which state this is streamed from.
> 
> @jkemp814 @MapGuy11 Any idea on the origin of this PBS ?

Could be a Satellite stream as Filmon used to do that.

BellezaEmporium committed:
Merge branch 'master' into LaneSh4d0w-patch-1

Dum4G committed:
Merge branch 'master' into LaneSh4d0w-patch-1

jkemp814 commented:
Let me check, I have a whole list of PBS

jkemp814 commented:
That one I do not believe it is a recent channel. They are streaming the Biden Inauguration, lol, as if it's happening right now.

MapGuy11 committed:
Merge branch 'master' into LaneSh4d0w-patch-1

Arupbaidya commented:
Star jalsa link

BellezaEmporium committed:
Merge branch 'master' into LaneSh4d0w-patch-1

LaneSh4d0w commented:
> 
> 
> That one, PBS, I do not believe it is a recent channel. They are streaming the Biden Inauguration, lol, as if it's happening right now.

Interesting. Weird thing.

Dum4G committed:
Merge branch 'master' into LaneSh4d0w-patch-1

spotanjo3 commented:
Are you going to update Portugal channels to Playlists by country Expand - >  https://iptv-org.github.io/iptv/countries/pt.m3u
? It is only 41 channels and some doesn't worked as I mentioned before. Keep up the good work!

Dum4G commented:
> Are you going to update Portugal channels

When we feel there's something to add

spotanjo3 commented:
> > Are you going to update Portugal channels
> 
> When we feel there's something to add

Thank you for replying. We reported that Sport TV 1 and 2 wasn't added yet and Sport TV 4 and 5 aren't working. They are aware of it but just wants you to know if you didn't know about it. Cheers!

Dum4G committed:
Update kw.m3u

Dum4G committed:
Merge branch 'master' into LaneSh4d0w-patch-1

allychikaponya commented:
is there iptv for Netflix. like mods apk

BellezaEmporium committed:
Merge branch 'master' into LaneSh4d0w-patch-1

BellezaEmporium committed:
Update in.m3u

LaneSh4d0w commented:
I'm going to add some Indian channels.
Kalaignar Murasu is an example of the latest additions.
![vlcsnap-2021-08-13-00h15m05s605](https://user-images.githubusercontent.com/30985701/129276884-81d108c3-6c35-47b2-8fff-a7bbed9734d1.png)

BellezaEmporium committed:
Update in.m3u

BellezaEmporium committed:
Update in.m3u

BellezaEmporium committed:
Update in.m3u

BellezaEmporium committed:
Update in.m3u

BellezaEmporium committed:
Update pk.m3u

BellezaEmporium committed:
Update in.m3u

BellezaEmporium committed:
Update in.m3u

BellezaEmporium committed:
Update in.m3u

BellezaEmporium committed:
Update in.m3u

BellezaEmporium committed:
Update in.m3u

Dum4G committed:
Update bg.m3u

LaneSh4d0w commented:
http://103.81.104.118/hls/stream3.m3u8 to look back later (still image)

BellezaEmporium committed:
Update us.m3u

BellezaEmporium committed:
Update us.m3u

BellezaEmporium committed:
Update in.m3u

BellezaEmporium committed:
Update in.m3u

BellezaEmporium committed:
Update in.m3u

BellezaEmporium committed:
Update in.m3u

BellezaEmporium committed:
Update in.m3u

BellezaEmporium committed:
Update bd.m3u

BellezaEmporium committed:
Update bd.m3u

BellezaEmporium committed:
Update bd.m3u

LaneSh4d0w commented:
@Dum4G I think it'll be good for me.

BellezaEmporium committed:
Update us.m3u

BellezaEmporium committed:
Merge branch 'LaneSh4d0w-patch-1' of https://github.com/iptv-org/iptv into LaneSh4d0w-patch-1

spotanjo3 commented:
`https://iptv-org.github.io/iptv/countries/pt.m3u`

It is not working right now

LaneSh4d0w commented:
> `https://iptv-org.github.io/iptv/countries/pt.m3u`
> 
> It is not working right now

Working fine for me.

![Screenshot_20210813-133211.jpg](https://user-images.githubusercontent.com/30985701/129351135-a9924a92-9bb1-4e50-8a30-90b3b3ab56e4.jpg)

spotanjo3 commented:
I am using VLC on my laptop. What do you use?

LaneSh4d0w commented:
I also use VLC. I sometimes use MPC-HC for verification purposes

Le ven. 13 août 2021 à 14:00, spotanjo3 ***@***.***> a écrit :

> I am using VLC on my laptop. What do you use?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/iptv-org/iptv/pull/4000#issuecomment-898408262>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHMM3ZLN7N3LEU7IWVFEACDT4UCN3ANCNFSM5BYUVICQ>
> .
>

spotanjo3 commented:
I see. I am on macOS.

allychikaponya commented:
is it possible to get Netflix or website on iptv or as iptv

Dum4G commented:
> is it possible to get Netflix or website on iptv or as iptv

nope

Dum4G committed:
Merge branch 'master' into LaneSh4d0w-patch-1
